### PR TITLE
Added endpoint to add votes to post

### PR DESCRIPTION
**Pull request by rlucaks**

### CODE REVIEW

File: apps/api/src/routes/post.ts
Comment (parfaxal) [inline apps/api/src/routes/post.ts:1]: authMiddleware?

File: apps/api/src/controllers/topic.ts [hunk 1]
```diff
@@ -20,9 +19,11 @@ class topicController {
     }
   }
   static async postsByTopicId(req: Request, res: Response, next: NextFunction) {
-    console.log(req.params["topicId"])
+    console.log(req.params["topicId"]);
```
Comment (DanDovi) [inline apps/api/src/controllers/topic.ts]: Do we still need this console log?

File: apps/api/src/controllers/topic.ts [hunk 2]
```diff
@@ -20,9 +19,11 @@ class topicController {
...
     try {
-      const result = await topicService.postsByTopicId({topicId: req.params["topicId"]});
+      const result = await topicService.postsByTopicId({
+        topicId: req.params["topicId"],
```
Comment (DanDovi) [inline apps/api/src/controllers/topic.ts]: I'd prefer that we use dot syntax if possible (`req.params.topicId`) - If typescript is giving errors then this is fine.

File: apps/api/src/routes/post.ts
Comment (DanDovi) [reply apps/api/src/routes/post.ts:1]: Looks like it's included on line 12

**Review by parfaxal — APPROVED** [review]

**Pull request merged**